### PR TITLE
FIX grafana service account creation and name

### PR DESCRIPTION
**Pull request by korjek**
## What does this PR change?

right now, service account of cost-analyzer is used in grafana deployment too. So there are two manifests with the same service account name and the last one wins.

## Does this PR rely on any other PRs?

No

## How does this PR impact users? (This is the kind of thing that goes in release notes!)

fixes service account for grafana

## What risks are associated with merging this PR? What is required to fully test this PR?

risk is really low.

## How was this PR tested?

 And this was tested locally.
Helm diff (`monitoring` is a release name)

```
ops, monitoring-kubecost-grafana, Deployment (apps) has changed:
  # Source: ops-monitoring/charts/cost-analyzer/templates/grafana-deployment.yaml
  apiVersion: apps/v1
  kind: Deployment
  metadata:
    name: monitoring-kubecost-grafana
    namespace: ops
    labels:
      app: grafana
      release: monitoring
      heritage: Helm
  spec:
    replicas: 1
    ...
      spec:
-       serviceAccountName: cost-analyzer
+       serviceAccountName: monitoring-kubecost-grafana
        securityContext:
          fsGroup: 1001
          fsGroupChangePolicy: OnRootMismatch
     ...
ops, monitoring-kubecost-grafana-clusterrolebinding, ClusterRoleBinding (rbac.authorization.k8s.io) has changed:
  # Source: ops-monitoring/charts/cost-analyzer/templates/grafana-clusterrolebinding.yaml
  kind: ClusterRoleBinding
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
    name: monitoring-kubecost-grafana-clusterrolebinding
    labels:
      app: grafana
      release: monitoring
      heritage: Helm
  subjects:
    - kind: ServiceAccount
-     name: cost-analyzer
+     name: monitoring-kubecost-grafana
      namespace: ops
  roleRef:
    kind: ClusterRole
    name: monitoring-kubecost-grafana-clusterrole
    apiGroup: rbac.authorization.k8s.io
ops, monitoring-kubecost-grafana, ServiceAccount (v1) has been added:
-
+ # Source: ops-monitoring/charts/cost-analyzer/templates/grafana-serviceaccount.yaml
+ apiVersion: v1
+ kind: ServiceAccount
+ metadata:
+   labels:
+     app: grafana
+     heritage: Helm
+     release: monitoring
+   name: monitoring-kubecost-grafana
+   namespace: ops
```

## Have you made an update to documentation? If so, please provide the corresponding PR.

Updated README.md

### CODE REVIEW

**Comment by korjek** [comment]
@chipzoller please take a look at this PR (instead of closed https://github.com/kubecost/cost-analyzer-helm-chart/pull/3068)

**Comment by chipzoller** [comment]
Thank you, must easier to parse. We'll take a look soon.

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -2550,6 +2550,10 @@ grafana:
   rbac:
     create: true
 
+  serviceAccount:
+    create: true
+    name:
```
Comment (cwoerner) [inline cost-analyzer/values.yaml]: Maybe make this `name: ""` to match the convention in other parts of this file.

Comment (chipzoller) [reply cost-analyzer/values.yaml]: Agree

**Comment by chipzoller** [comment]
Can you pull in latest changes, please? I'm unable to do so for you.

**Comment by korjek** [comment]
> Can you pull in latest changes, please? I'm unable to do so for you.

Done!

**Review by cwoerner — APPROVED** [review]

**Comment by korjek** [comment]
@chipzoller is there any estimation of when new release will be available? This is a blocker for us as IRSA is broken because of this bug

**Comment by jessegoodier** [comment]
> Can you pull in latest changes, please? I'm unable to do so for you.

This looks good to me and useful. any concerns @chipzoller ?

**Comment by chipzoller** [comment]
I think it'd be fine but I'm slightly concerned about upgrades. Haven't had the time to test this yet.

**Comment by korjek** [comment]
@chipzoller could we have this released finally please?

P.S. I can't imagine how ppl keep their IRSA working atm. Any hints?

**Comment by chipzoller** [comment]
Sorry for the delay. If you can resolve these conflicts and merge `develop`, I'll test and get this in right away.

**Comment by cwoerner** [comment]
> Sorry for the delay. If you can resolve these conflicts and merge `develop`, I'll test and get this in right away.

Thanks Chip!  I resolved these.  Volodymyr will review on Monday if there are any issues arising from the merge.

**Comment by cwoerner** [comment]
> > Sorry for the delay. If you can resolve these conflicts and merge `develop`, I'll test and get this in right away.
> 
> Thanks Chip! I resolved these. Volodymyr will review on Monday if there are any issues arising from the merge.

(It's not obvious, but Volodymyr and I work together)

**Review by chipzoller — APPROVED** [review]
Don't see any problems with a basic installation of 2.1.0 and upgrade using these values. Approving.

**Comment by chipzoller** [comment]
We'll leave for 2.2 on second thought. It should be coming this month.

**Pull request merged**